test/e2e/kubectl/kubectl.go

```diff
@@ -1676,7 +1676,12 @@ metadata:
 			// we expect following values for: Major -> digit, Minor -> numeric followed by an optional '+',  GitCommit -> alphanumeric
 			requiredItems := []string{"Client Version: ", "Server Version: "}
 			for _, item := range requiredItems {
-				if matched, _ := regexp.MatchString(item+`v\d\.\d+\.[\d\w\-\.\+]+`, versionString); !matched {
+				// prior to 1.28 we printed long version information
+				oldMatched, _ := regexp.MatchString(item+`version.Info\{Major:"\d", Minor:"\d+\+?", GitVersion:"v\d\.\d+\.[\d\w\-\.\+]+", GitCommit:"[0-9a-f]+"`, versionString)
```

Comment:
We can not run `kubectl version -ojson` because kubectl is still old and will fail?.

Reply:
we could, but I don't want that, I want the bare `kubectl version` output here, only